src/sgame/sg_weapon.cpp

```diff
@@ -169,10 +169,12 @@ bool G_RefillAmmo( gentity_t *self, bool triggerEvent )
 	if ( BG_Weapon( self->client->ps.stats[ STAT_WEAPON ] )->maxClips > 0 )
 	{
 		GiveMaxClips( self );
+		GiveFullClip( self );
 
 		if ( triggerEvent )
 		{
 			G_AddEvent( self, EV_CLIPS_REFILL, 0 );
+			G_AddEvent( self, EV_AMMO_REFILL, 0 );
```

Comment:
This makes two instances of the same sound play at once. Just delete EV_CLIPS_REFILL from the enum.

Reply:
Fixed